terraform/deploy-dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+TERRAFORM_DIR="$(dirname "$0")"
+
+accountId="$(aws sts get-caller-identity | jq -r .Account)"
+region=$AWS_REGION
+
+imageRepo="$accountId.dkr.ecr.$region.amazonaws.com/notify"
+
+aws ecr get-login-password --region eu-central-1 | docker login --username AWS --password-stdin "$imageRepo"
+# --platform=linux/amd64: Must target linux/amd64 as that is what ECS runs.
+docker build "$TERRAFORM_DIR/.." -t "$imageRepo" --build-arg=release=true --platform=linux/amd64 $BUILD_ARGS
+imageVersion="$(docker inspect --format="{{ .Id }}" "$imageRepo" | cut -d: -f2)"
+tag="$imageRepo:$imageVersion"
+docker tag "$imageRepo" "$tag"
+docker push "$tag"
+
+# TF_VAR_* env vars not supported for remote deployments, so use *.auto.tfvars instead which works
+autoTfVars="$TERRAFORM_DIR/dev.auto.tfvars"
```

Comment:
Can you make sure this file is deleted when the script is done otherwise it will be active for subsequent plans/applies regardless of the workspace.

Reply:
Added here: 5d770ef